docs/en/configuration.rst

```diff
@@ -171,7 +171,7 @@ specified under the ``environments`` nested collection. For example:
             pass: ''
             port: 3306
             charset: utf8
-            collation: utf8_unicode_ci
+            collation: utf8mb4_general_ci
```

Comment:
Is there a reason not to keep unicode? At least that's the default I am usually using.

Reply:
cakephp defaults to utf8mb4 if that makes a difference here.

Reply:
Lines above should be 
```yaml
charset: utf8mb4
collation: utf8mb4_unicode_ci
```

See https://stackoverflow.com/a/766996/4228563.

`utf8mb4` supports more characters (e.g. emojis) than `utf8`. 

`unicode` is more universal for sorting and comparison in wide range of languages than `general`.

Reply:
Agree that unicode is best